### PR TITLE
added discord widget

### PR DESCRIPTION
**Pull request by Jolissa1**

### CODE REVIEW

**Comment by DrKlonk** [comment]
image
On a wide screen, the space between the text and widget is larger than the space to the section above, making it feel a little detached.

File: frontend/pages/index.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@
         </ul>
       </section>
     </BaseContainer>
-    <BaseContainer class="mission" containerType="narrow">
+    <BaseContainer class="mission" containerType="narrow" :flexCol="true">
```
Comment (DrKlonk) [inline frontend/pages/index.vue:20]: This one isn't necessary

File: frontend/pages/index.vue [hunk 2]
```diff
@@ -39,6 +39,29 @@
         >
       </div>
     </BaseContainer>
+    <BaseContainer class="discord__container">
+      <div class="discord__container-text">
+        <h2>Join Our Discord Community</h2>
```
Comment (DrKlonk) [inline frontend/pages/index.vue:44]: I miss a BEM class and a bit more space below (like in the our mission). They should be consistent at least

File: frontend/pages/index.vue [hunk 3]
```diff
@@ -132,4 +156,27 @@ export default defineComponent({
     }
   }
 }
+
+.discord__container.container {
```
Comment (DrKlonk) [inline frontend/pages/index.vue:160]: MINOR: The .container isn't really needed, which would make it possible to use the "&"

File: frontend/pages/index.vue [hunk 4]
```diff
@@ -132,4 +156,27 @@ export default defineComponent({
...
+  .discord__container-text {
+    p {
+      margin-top: 1rem;
+      width: 60%;
+    }
+  }
+  @include respond(tab-portrait) {
```
Comment (DrKlonk) [inline frontend/pages/index.vue:167]: image
It's not really responsive like this

**Review by DrKlonk — APPROVED** [review]

File: frontend/pages/index.vue (hunk 3 above)
Comment (Jolissa1) [reply frontend/pages/index.vue:160]: done

File: frontend/pages/index.vue (hunk 2 above)
Comment (Jolissa1) [reply frontend/pages/index.vue:44]: changed

File: frontend/pages/index.vue (hunk 3 above)
Comment (DrKlonk) [reply frontend/pages/index.vue]: Actually, it is a bit weird naming wise. It think the main class shouldn't be a block. Maybe just 'discord' as a main class and then __container, __text, __heading and __paragraph to keep it consistent?

Comment (Jolissa1) [reply frontend/pages/index.vue]: changed

File: frontend/pages/index.vue [hunk 5]
```diff
@@ -157,18 +157,27 @@ export default defineComponent({
...
+.discord__container {
+  &-title {
+    margin-bottom: 2rem;
+    margin-right: 1rem;
+  }
+  &-text {
     p {
-      margin-top: 1rem;
-      width: 60%;
+      width: 80%;
```
Comment (DrKlonk) [inline frontend/pages/index.vue:167]: image

This makes it inconsistent with the section above. This is not to say that that one is better, but it looks a bit strange.

It's also just about 75ch wide on the widest screens, which is on the edge of nicely readable.

**Review by DrKlonk — APPROVED** [review]

**Pull request merged**